lib/roman_converter.js

```diff
@@ -1,27 +1,32 @@
+
+
 const convertToOldRoman = (number) => {
-  let romanNumerals = '';
-  let decimal = [1000, 500, 100, 50, 10, 5, 1];
-  let numerals = ["M", "D", "C", "L", "X", "V", "I"];
-  for (let i = 0; i<=decimal.length; i++) {
-    while (number % decimal[i] < number) {
-      romanNumerals += numerals[i];
-      number -= decimal[i];
+  var result = '';
```

Comment:
@zacharylandes - any reason you are using `var` as opposed to `let` (for line 4) and `const` (for lines 5-6)?